inst/extras/build.sh

```diff
@@ -0,0 +1,7 @@
+~/bin/R-4.0.0/bin/R CMD build ../../ --resave-data #--no-examples  --no-build-vignettes
+~/bin/R-4.0.0/bin/R CMD check mia_0.0.0.9010.tar.gz #--no-build-vignettes --no-examples
+~/bin/R-4.0.0/bin/R CMD BiocCheck mia_0.0.0.9010.tar.gz
+~/bin/R-4.0.0/bin/R CMD INSTALL mia_0.0.0.9010.tar.gz
+
+
+
```

Comment:
This build script we do not necessarily need to have in github; I am not sure if it matters for others.

Reply:
Deleted